usr/lib/cca_stdll/cca_specific.c

```diff
@@ -1446,6 +1551,9 @@ static CK_RV cca_iterate_domains(STDLL_TokData_t *tokdata, const char *device,
 
             domain = cca_private->usage_domains[i];
         }
+        else {
+            domain = 0;
```

Comment:
this code here is certainly not wrong, but the `domain` has already been assigned above by `cca_get_current_domain()` if `cca_private->dom_any` is FALSE.